### PR TITLE
[#155] add DB upgrade command to Dockerfile and manage.py; update REA…

### PR DESCRIPTION
**Pull request by adam-kaplan-thoughtworks**
…DME to clarify purpose of SQL file on Dropbox

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -48,8 +48,7 @@ CMD initdb; pg_ctl -D /var/lib/postgresql/data start; createdb; echo "create use
     echo "CREATE DATABASE invisible_flow_testing;" | psql; echo "GRANT ALL PRIVILEGES ON DATABASE invisible_flow_testing TO invisible_flow_testing;" | psql; \
     echo "CREATE EXTENSION postgis;" | psql -U invisible_flow_testing; \
     service postgresql start; \
+    python3 manage.py db upgrade --directory invisible_flow/migrations; \
```
Comment (adam-kaplan-thoughtworks) [inline Dockerfile:51]: This command should replace the deleted one below it (#54)

File: Dockerfile [hunk 2]
```diff
@@ -48,8 +48,7 @@ CMD initdb; pg_ctl -D /var/lib/postgresql/data start; createdb; echo "create use
...
     echo "password" | su invisible_flow_testing; \
-    echo "ALTER TABLE data_allegation ADD COLUMN cr_id character varying(50);" | psql -U invisible_flow_testing; \
```
Comment (adam-kaplan-thoughtworks) [inline Dockerfile:52]: no longer needed; coming from migrations

File: manage.py [hunk 3]
```diff
@@ -66,3 +67,10 @@ def setup_db(_app: Flask) -> SQLAlchemy:
 db = setup_db(app)
 
 migrate = Migrate(app, db)
+
+manager = Manager(app)
```
Comment (adam-kaplan-thoughtworks) [inline manage.py:71]: some magic to allow commands that were added to Dockerfile to come to life.

File: readme.md [hunk 4]
```diff
@@ -34,20 +34,15 @@ Let **npm** do its thing. Once complete run:
 
 To make sure that the frontend built successfully, check the `frontend` directory for a new folder named `build` and note that it also updates the `node_modules` folder.
 
-### Add `invisible-flow-dump.sql` to your root directory
-The presence of the `invisible-flow-dump.sql` file in our project ensures that all contributors have the same schema and default data imported into their database when building the app with docker. 
+### Download CPDP Database
```
Comment (adam-kaplan-thoughtworks) [inline readme.md:37]: This part of the readme could benefit from some rewording. The copy of CPDP doesn't need to be present in the project in order to run, and probably shouldn't be lest it be committed.

**Comment by adam-kaplan-thoughtworks** [comment]
repro steps:
1. clone this project's repository into a new directory on your computer
2. `cd` into that directory, `git checkout ak/155` so you're using this branch's code, and go through the onboarding instructions in the readme up to "check if the app is running correctly"
3. "check if the app is running correctly" instructions might warrant some rewriting, but once you get to this section, you will want to visit `localhost:5000`, run the scraper, and watch the database fill with things by performing the commands suggested in the current version of the readme.

expected:
database has records

I was able to get thru testing but had to backtrack and do things in the readme that i didn't do the first time. I also got tripped up by running Docker from the wrong directory, but that is likely an Adam specific problem.

**Comment by reblg** [comment]
@adam-kaplan-thoughtworks can you actually set up some time with me to go over this so i know exactly what needs to be changed in the readme based on these changes.

**Comment by adam-kaplan-thoughtworks** [comment]
All good after a short convo

**Pull request merged**